README.MD

```diff
@@ -14,7 +14,7 @@ Here are some resources you should take a look at:
 
 - [The wiki](https://github.com/gbdev/pandocs/wiki), containing various rules and contribution guidelines you will have to follow when proposing changes. Some RFCs and discussions detailing the document scope are linked there as well.
 - [The project board](https://github.com/gbdev/pandocs/projects/1), for a general picture of the roadmap and the ongoing efforts.
-- [The issues](https://github.com/gbdev/pandocs/issues), where we discuss what needs to be worked on, and how.
+- [The issues](https://github.com/gbdev/pandocs/issues), where we discuss what needs to be worked on, and how. If you are looking into investigating Game Boy unknown behaviours, check the Issues labeled with ["research"](https://github.com/gbdev/pandocs/issues?q=is%3Aissue+is%3Aopen+label%3Aresearch).
```

Comment:
Should we document more of the labels? Also, maybe splitting this info in a `CONTRIBUTING.md` file?

Reply:
I think it's okay to just mention *some* of them here and everything in the CONTRIBUTING file. On this topic: https://github.com/gbdev/pandocs/issues/370